code/src/components/AddTask.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/tasks'
+import styled from 'styled-components'
+
+
+export const AddTaskForm = () => {
+  const [todo, setTodo] = useState('')
+  const dispatch = useDispatch()
+
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(tasks.actions.addTask(todo))
+    setTodo('')
+  }
+
+  return (
+    <FormWrapper onSubmit={handleSubmit}>
+      <label>
+        <Input type="text" multiline={true} maxLength={20} placeholder="New Todo" value={todo} onChange={(event) => setTodo(event.target.value)} />
+      </label>
+      <Button type="submit">Add task</Button>
+    </FormWrapper>
+  )
+}
+
+const FormWrapper = styled.form`
+  display: flex;
+  justify-content: space-around; 
+  padding: 10px; 
+  background-color: #cfcfcf;
+`
```

Comment:
Great job using the styled components, apparently it's more "correct" to create these on top of the react stuff. So first you import whatever you need in the file, then you setup the styled components then you create the react component.
This works as well, but it's more logical to set them up before using them.